code/datums/status_effects/xeno_buffs.dm

```diff
@@ -161,7 +162,7 @@
 	var/mob/living/carbon/xenomorph/heal_target
 	var/heal_amount
 
-	if(amount >= 0)
+	if(amount > -DRONE_BASE_SALVE_HEAL) // lowest heal in the game
```

Comment:
Why ?

Reply:
This was already changed in the latest commit to be a **-60**. This is because, after extensive testing and confirmation, most forms of passive regeneration do not go past the value of 60 (after bonuses such as recovery aura, resting weeds, etc).

Reply:
I see you actually checked the latest commit, so I'm resolving this.